util/kustomize/kustomize.go

```diff
@@ -63,6 +79,61 @@ func (k *kustomize) Build(opts KustomizeBuildOpts, overrides []*v1alpha1.Compone
 
 	return objs, append(getImageParameters(objs)), nil
 }
+func (k *kustomize) GetCommandName() (string, error) {
+
+	version, err := k.getKustomizationVersion()
+	if err != nil {
+		return "", err
+	}
+
+	log.Infof("using kustomize version=%d", version)
+
+	if version == 1 {
+		return "kustomize", nil
+	} else {
+		return "kustomize" + strconv.Itoa(version), nil
+	}
+}
+
+// kustomization is a file called kustomization.yaml that describes a configuration consumable by kustomize.
+func (k *kustomize) findKustomization() (string, error) {
+	for _, file := range []string{"kustomization.yaml", "kustomization.yml", "Kustomization"} {
```

Comment:
this function will also need to be updated:
* `IdentifyAppSourceTypeByAppDir`
* repository.go::func (s *Server) listAppsPaths()